### DISCUSSION TITLE
[4.x] Contact Custom Fields

### DISCUSSION THREAD

**Discussion by leoalv** [joomla/joomla-cms#36944] [General]
We have used extra fields in contacts of type contact and Mail, we have migrated the sites, but since Joomla 4 there is no link to access the fields for contacts as in Joomla 3.

They removed these accesses because they intend to eliminate extra fields for contacts?

If they are going to eliminate the extra fields, they must notify us or they must create the direct accesses.

**Comment by brianteeman** [comment]
They are still there.

Do you not have these links on the side menu

### Sidebar
![image](https://user-images.githubusercontent.com/1296369/152674172-1d3f16ce-1c90-4c56-a857-92277a2ece84.png)

If you do not then please check the contact component options in the integration tab 

### Options
![image](https://user-images.githubusercontent.com/1296369/152674199-5987bacc-3b63-4760-862e-06b1c53f6f77.png)

**Reply by leoalv** [reply]
@brianteeman Thanks for your reply, but after many tests I have discovered that the problem is not with the contacts component. The problem is that the Joomla migrator from version 4.0.4 onwards does not make a correct migration of REPEATABLE fields. I had migrated repeatable fields to Joomla 4.0.3 without any problem, so today I tried manual migrations with J4.0.0 J4.0.1 J4.0.2 and J4.0.3 and it works with all of them except J4. 0.4 onwards.

It is a bug of Joomla migrator?

**Reply by web-tiki** [reply]
@leoalv I have the same issue with the contact custom fields disapearing when migrating to 4.2.3
Have you found a solution ?

**Reply by leoalv** [reply]
@web-tiki As my previous comment says, if your fields are repeatable, do not migrate directly to Joomla 4.2.3, you have to migrate to Joomla 4.0.3 and then update, possibly to Joomla 4.1 and from there to the latest version. These are the steps I follow now if I want to migrate custom fields.
